microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -46,18 +46,18 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
-
- > [!IMPORTANT]
-> You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
+1. The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
```

Comment:
```suggestion
   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
```
in Step > Result format, number the action steps that the user does, not the results.